libs/blocks/table/table.js

```diff
@@ -34,16 +34,18 @@ function handleHeading(headingCols, isPriceBottom) {
       col.innerHTML = `<p class="tracking-header">${col.innerHTML}</p>`;
     } else {
       let textStartIndex = 0;
-      if (elements[0]?.querySelector('img')) {
+      const iconTile = elements[0]?.querySelector('img');
+      if (iconTile) {
         textStartIndex += 1;
+        if (!(table.classList.contains('merch'))) iconTile.closest('p').classList.add('header-product-tile');
       }
       elements[textStartIndex]?.classList.add('tracking-header');
       const pricingElem = elements[textStartIndex + 1];
       let bodyElem = elements[textStartIndex + 2];
 
       if (pricingElem) {
         pricingElem.classList.add('pricing');
-        if (isPriceBottom) {
+        if (table.classList.contains('pricing-bottom')) {
```

Comment:
nit: since `table.classList.contains('pricing-bottom')` is used twice, assign to a var

Reply:
@chrischrischris I have made the suggested change.